docs/docusaurus/docs/core/data_quality/01_schema.md

```diff
@@ -0,0 +1,204 @@
+## Data Quality: Ensuring Schema Consistency with Great Expectations
+
+When working with data, you might face common challenges that necessitate a data quality platform.
```

Comment:
I think your content for the opening paragraph is good, but I'd like to see the definition of data schema expressed more clearly and succinctly upfront vs. woven into the message of why it's important to validate. (We definitely want to keep the explanation of why it's important - but I'd like the content to distinguish the _what_ and the _why_ more distinctly for the user).

For each of the data quality dimensions that we create a use case for, I think it will be helpful to the user to begin with a concise understanding of "What is schema?"/"What is missingness?"/"What is <insert data quality dimension here>?"

Reply:
we were aiming for "show, don't tell", but including a concise explanation upfront would add clarity